javascript/linked-list/README.md

```diff
@@ -60,3 +60,31 @@
 
 ## API
 <!-- Description of each method publicly available to your Linked List -->
+
+
+# Code challenge 06
+
+## Challenge Summary
+### Feature Tasks
+Write the following methods for the Linked List class:
+
+- append
+  - arguments: new value
+  - adds a new node with the given value to the end of the list
+- insert before
+  - arguments: value, new value
+  - adds a new node with the given new value immediately before the first node that has the value specified
+- insert after
+  - arguments: value, new value
+  - adds a new node with the given new value immediately after the first node that has the value specified
+
+## Whiteboard Process
+<!-- Embedded whiteboard image -->
+
+## Approach & Efficiency
+<!-- What approach did you take? Why? What is the Big O space/time for this approach? -->
+
+## Solution
+<!-- Show how to run your code, and examples of it in action -->
+
+## Collaborators: Martha QR,  Tony R. , Luis S. , Zayah P 
```

Comment:
This is it own readme. For the readme it one per challenge for example you can call this one readme-06.md(or whatever you want to call it) that way it easier to find the information instead of over writing it each time you do a new challenge.